src/qibo/noise.py

```diff
@@ -1,4 +1,32 @@
 from qibo import gates
+from qibo.config import raise_error
+
+
+class CustomError:
+    """Quantum error associated with the :class:`qibo.gates.Channel`
+
+    Args:
+        channel (:class:`qibo.gates.Channel`): any channel
+
+    Example:
+
+    .. testcode::
+
+        import numpy as np
+        from qibo.gates import KrausChannel
+        from qibo.noise import Error
```

Comment:
If you called the class `CustomError` also here you should use `CustomError` instead of `Error`.